src/applications/simple-forms/40-0247/tests/e2e/0247-pmc.cypress.spec.js

```diff
@@ -34,100 +75,49 @@ const testConfig = createTestConfig(
             .click();
         });
       },
-      [pagePaths.veteranPersonalInfoPage]: ({ afterHook }) => {
+      ...pageTestConfigs,
+      'applicant-address': ({ afterHook }) => {
         afterHook(() => {
           cy.get('@testData').then(data => {
-            const {
-              veteranFullName,
-              veteranDateOfBirth,
-              veteranDateOfDeath,
-            } = data;
-
-            fillFullNameWebComponentPattern('veteranFullName', veteranFullName);
-            fillDateWebComponentPattern(
-              'veteranDateOfBirth',
-              veteranDateOfBirth,
-            );
-            fillDateWebComponentPattern(
-              'veteranDateOfDeath',
-              veteranDateOfDeath,
-            );
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
+            cy.get('input[name="root_applicantAddress_state"]')
+              .should('not.have.attr', 'disabled')
```

Comment:
here

Reply:
Yeah, I think the page-specific test-block here is the right place to assert on a specific field.  I've also removed the tagname (or element-type) from my selector above, so the assertion works on both text-inputs & selects.